Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonAttributePanel.java

```diff
@@ -587,76 +667,91 @@ private void cancelButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-
-        handleIntraCaseSearchCriteriaChanged();
     }//GEN-LAST:event_intraCaseRadioActionPerformed
 
-    public void handleIntraCaseSearchCriteriaChanged() {
-        if (this.areIntraCaseSearchCriteriaMet()) {
-            this.searchButton.setEnabled(true);
-            this.hideErrorMessages();
-        } else {
-            this.searchButton.setEnabled(false);
-            this.hideErrorMessages();
-            this.showIntraCaseErrorMessage();
-        }
-    }
-
     private void interCaseRadioActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_interCaseRadioActionPerformed
         ((java.awt.CardLayout) this.layoutPanel.getLayout()).last(this.layoutPanel);
-        handleInterCaseSearchCriteriaChanged();
     }//GEN-LAST:event_interCaseRadioActionPerformed
 
     private void formWindowClosed(java.awt.event.WindowEvent evt) {//GEN-FIRST:event_formWindowClosed
         SwingUtilities.windowForComponent(this).dispose();
     }//GEN-LAST:event_formWindowClosed
 
-    public void handleInterCaseSearchCriteriaChanged() {
-        if (this.areInterCaseSearchCriteriaMet()) {
-            this.searchButton.setEnabled(true);
-            this.hideErrorMessages();
+    private void percentageThresholdCheckActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_percentageThresholdCheckActionPerformed
+        if (this.percentageThresholdCheck.isSelected()) {
+            this.percentageThreshold.setEnabled(true);
         } else {
-            this.searchButton.setEnabled(false);
-            this.hideErrorMessages();
-            this.showInterCaseErrorMessage();
+            this.percentageThreshold.setEnabled(false);
+        }
+
+        this.handleFrequencyPercentageState();
+    }//GEN-LAST:event_percentageThresholdCheckActionPerformed
+
+    private void percentageThresholdChanged(){
+        String percentageString = this.percentageThreshold.getText();
+
+        try {
+            this.percentageThresholdValue = Integer.parseInt(percentageString);
+            
+        } catch (NumberFormatException exception) {
+            this.percentageThresholdValue = -1;
         }
+
+        this.handleFrequencyPercentageState();        
     }
+    
+    private void updateErrorTextAndSearchBox() {
 
-    private void toggleErrorTextAndSearchBox() {
-        if (!this.pictureVideoCheckbox.isSelected() && !this.documentsCheckbox.isSelected() && !this.allFileCategoriesRadioButton.isSelected()) {
+        if (this.errorManager.anyErrors()) {
             this.searchButton.setEnabled(false);
+            //grab the first error error and show it
+            this.errorText.setText(this.errorManager.getErrors().get(0));
             this.errorText.setVisible(true);
         } else {
             this.searchButton.setEnabled(true);
             this.errorText.setVisible(false);
         }
     }
 
-    private void manageCheckBoxState() {
+    private void enablePercentageOptions() {
```

Comment:
In this and the disable section, you need to add the second half of the text (jLabel1)